### PR TITLE
I41 study room email

### PR DESCRIPTION
**Pull request by carolyncole**
Waiting on April to see what to do about the blank expiration.  I believe we will set it on assignment creation, which will make it show in the email.  I'm putting in a ticket so this work can move forward.
Screen Shot 2021-12-07 at 3 48 49 PM

### CODE REVIEW

File: app/views/user_mailer/study_room_assignment_confirmation.text.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<%= @study_room_assignment.general_area %> Assignment
+<%= @study_room_assignment.name %> is assigned to <%= @study_room_assignment.location %>
+Occupency expires on <%= @study_room_assignment.expiration_date %>
```
Comment (christinach) [inline app/views/user_mailer/study_room_assignment_confirmation.text.erb:3]: Occupancy

**Review by christinach — COMMENTED** [review]
@carolyncole one small comment.

File: app/views/user_mailer/study_room_assignment_confirmation.text.erb (hunk 1 above)
Comment (carolyncole) [reply app/views/user_mailer/study_room_assignment_confirmation.text.erb:3]: Thanks @christinach!

**Comment by carolyncole** [comment]
@christinach this should be fixed up.

**Review by christinach — APPROVED** [review]

**Pull request merged**